tests/countries/test_eritrea.py

```diff
@@ -0,0 +1,149 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import GOVERNMENT
+from holidays.countries.eritrea import Eritrea, ER, ERI
+from tests.common import CommonCountryTests
+
+
+class TestEritrea(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1994, 2050)
+        super().setUpClass(Eritrea, years=years)
+        cls.government_holidays = Eritrea(categories=GOVERNMENT, years=years)
+        cls.no_estimated_holidays = Eritrea(years=years, islamic_show_estimated=False)
+
+    def test_country_aliases(self):
+        self.assertAliases(Eritrea, ER, ERI)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Eritrea(years=1993))
+
+    def test_new_years_day(self):
+        self.assertHolidayName("New Year's Day", (f"{year}-01-01" for year in range(1994, 2050)))
+
+    def test_orthodox_christmas(self):
+        self.assertHolidayName(
+            "Orthodox Christmas", (f"{year}-01-07" for year in range(1994, 2050))
+        )
+
+    def test_epiphany(self):
+        self.assertHolidayName(
+            "Epiphany",
+            (f"{year}-01-19" for year in range(1994, 2050) if year % 4 != 0),
+            (f"{year}-01-20" for year in range(1994, 2050) if year % 4 == 0),
+        )
+
+    def test_orthodox_easter(self):
+        name = "Orthodox Easter"
+        dt = (
+            "2020-04-19",
+            "2021-05-02",
+            "2022-04-24",
+            "2023-04-16",
+            "2024-05-05",
+            "2025-04-20",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, self.no_estimated_holidays, range(1994, 2050))
+
```

Comment:
Missing tests for Good Friday and International Workers' Day; test methods should be arranged according to holidays order.